Automattic/wp-calypso
# VideoPress TV Signup Flow

Related to https://github.com/Automattic/greenhouse/issues/1776

## Proposed Changes

* Introduces the new VideoPress tv signup and purchase flows.

## Testing Instructions



* test both logged in and not logged in
* on calypso.localhost, visit `http://calypso.localhost:3000/setup/videopress-tv`. Or the equivalent in Calypso-live.
* after an optional login, you are redirected to your new site.
* Try it again with the same user, you should end up in the same site (one trial per user)
* Test purchasing a plan for the site (using `http://calypso.localhost:3000/setup/videopress-tv-purchase/?siteSlug=YOUR_SITE_SLUG`). When purchased the trial "nag" screen should go away.
* Try to create a new videopress TV site: you are able to create one again since your previous one has an active plan.

## Pre-merge Checklist
